bedrock/mozorg/templates/mozorg/about/leadership.html

```diff
@@ -670,6 +670,49 @@ <h3 class="group-title" id="mgmt-corp"><a href="{{ url('foundation.moco') }}">{{
         </div>
       </article>
 
+      <article id="alan-davidson" data-id="alan-davidson" class="vcard has-bio" itemscope itemtype="http://schema.org/Person">
+        <figure class="headshot">
+          {{ high_res_img('mozorg/about/leadership/alan-davidson.jpg', {'alt': '', 'width': '160', 'height': '160', 'class': 'photo', 'itemprop': 'image'}) }}
+          <figcaption><h4 class="fn" itemprop="name">Alan Davidson</h4></figcaption>
+        </figure>
+
+        <div class="person-info">
+          <p class="title" itemprop="jobTitle">{{ _('VP of Global Policy, Trust and Security') }}</p>
+          <p class="since">{{ _('With Mozilla since:') }} 2018</p>
+
+          <ul class="utility">
+            <li><a class="file" rel="external" href="https://assets.mozilla.net/leadership/mozilla_portrait_Chris-Beard.jpg">{{ _('Download high resolution photo') }}</a></li>
```

Comment:
Should this be Alan Davidson's portrait?

Reply:
D'oh! Hazards of copy-paste. Will update.